config/packages/security.yaml

```diff
@@ -45,7 +45,7 @@ security:
     # Easy way to control access for large sections of your site
     # Note: Only the *first* access control that matches will be used
     access_control:
-        - { path: '^/api', roles: ROLE_ADMIN }
+        #- { path: '^/api', roles: ROLE_ADMIN }
```

Comment:
you have to redo your access control list properly
let everybody register, let everybody login, let only admins delete a user etc